include/capo/sound.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+#include <capo/types.hpp>
+#include <capo/utils/id.hpp>
+
+namespace capo {
+class Instance;
+
+///
+/// \brief Lightweight handle to a ready-to-play audio clip, mounted on device-accessible memory; use Instance to create
+///
+class Sound {
+  public:
+	Sound() = default;
+	static Sound const blank;
```

Comment:
Can't this be inline & constexpr? Same goes for Source::blank

Reply:
Not inside the class, the definition isn't complete yet. So you'd need to disambiguate via prefixes in the name, etc.